community/codelabs/docai-ocr/batch_processing_directory.py

```diff
@@ -0,0 +1,117 @@
+# type: ignore[1]
+# pylint: skip-file
+"""
+Makes a Batch Processing Request to Document AI
+Creates request with full directory in Cloud Storage
+"""
+import re
+from typing import List
+
+from google.api_core.client_options import ClientOptions
+from google.cloud import documentai_v1 as documentai
+from google.cloud import storage
+
+
+PROJECT_ID = "YOUR_PROJECT_ID"
+LOCATION = "YOUR_PROJECT_LOCATION"  # Format is 'us' or 'eu'
+PROCESSOR_ID = "YOUR_PROCESSOR_ID"  # Create processor in Cloud Console
+
+# Format 'gs://input_bucket/directory'
+GCS_INPUT_PREFIX = "gs://cloud-samples-data/documentai/codelabs/ocr/multi-document"
```

Comment:
same question is this publicly available

Reply:
Yes, allUsers have read access